src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogAgentClient.java

```diff
@@ -320,6 +481,32 @@ private boolean reinitializeAgentHttpClient(boolean force) {
         }
     }
 
+    protected boolean checkEvpProxySupportAndUpdateLogic() {
+        if (evpProxySupported) {
+            return true; // Once we have seen an Agent that supports EVP Proxy, we never check again.
```

Comment:
Could we have the case of customers downgrading the DD Agent?

Reply:
I decided to not support that case so we don't have to always query `/info`.